examples/gnrc_tftp/Makefile

```diff
@@ -0,0 +1,42 @@
+# name of your application
+APPLICATION = gnrc_tftp
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+BOARD_INSUFFICIENT_MEMORY := airfy-beacon chronos msb-430 msb-430h nrf51dongle \
```

Comment:
Try `make buildtest` and check for the output. If the error says something like `cannot move location counter backwards`, it's probably a memory related issue.